lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -110,6 +110,44 @@ def self.run(params)
 
           Helper::RevenuecatInternalHelper.create_pr(pr_title, body, repo_name, current_branch, new_branch_name, github_pr_token, [label])
         end
+
+        # For releases older than the latest published version, we need to create a new PR against main to update the changelog with the changes from this release
+        is_older_than_latest_published_version = !Helper::RevenuecatInternalHelper.newer_than_latest_published_version?(new_version_number)
+        if is_older_than_latest_published_version
+
+          if dry_run
+            Helper::RevenuecatInternalHelper.discard_changes_in_current_branch
+          else
+            UI.important("Creating a new PR against main to update the changelog with the changes from this release")
+          end
+
+          main_branch = "main"
+          Helper::RevenuecatInternalHelper.create_or_checkout_branch(main_branch)
```

Comment:
Hmm TBH, I'm wondering if we should split this into its own action, so we can call it separately from the rest of the bump action (possibly in a separate job?)... that way we don't need to explicitly checkout main here nor cleanup previous leftovers from the process... Wdyt? 

This could also be moved as part of the release process, instead of the creation of the PR, in case we recreate the release PR multiple times for any reason. So only do this once a release is actually out.

Reply:
That's a very good point. I did it tis way because it followed the same idea of opening the release PR before actually making the release. But it's true that making it its own separate action that happens right after a release is much cleaner. 
Thanks for the suggestion!